src/ceph-volume/ceph_volume/devices/lvm/batch.py

```diff
@@ -269,6 +270,17 @@ def __init__(self, argv):
                   ' if more slots then osds-per-device are specified, slots'
                   'will stay unoccupied'),
         )
+        def data_allocate_fraction(pct):
```

Comment:
Could you move this function to `utils.arg_validators` please? You can see an example usage with `arg_validators.ValidDevice` here: https://github.com/ceph/ceph/pull/40659/files#diff-37d41dc2d64315970e0a32c6d84be634f82c62a15c4e3144020ea2be88641601R200